open_academy/wizards/open_academy_wizard_views.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="open_academy_wizard_view_form" model="ir.ui.view">
+        <field name="name">open.academy.wizard.view.form</field>
+        <field name="model">open.academy.wizard</field>
+        <field name="arch" type="xml">
+            <form>
+                <field name="session_wizard_ids" string="Session"/>
+                <field name="attendee_wizard_ids" string="Attendees"/>
```

Comment:
Same here. This fields do not exists anymore.

Please test your changes before ask for revision.